.github/workflows/issue-opened.yml

```diff
@@ -93,9 +93,9 @@ jobs:
           issue_number: ${{ github.event.issue.number }}
           openai_api_key: ${{ env.AUTOTRIAGER_OPENAI_API_KEY }}
           add_labels: true
-          labels_file: ${{ github.workspace }}/.github/auto-triager/labels.txt
-          types_file: ${{ github.workspace }}/.github/auto-triager/types.txt
-          prompt_file: ${{ github.workspace }}/.github/auto-triager/prompt.txt
+          labels_file: ${{ github.workspace }}/.github/workflows/auto-triager/labels.txt
```

Comment:
point to here https://github.com/grafana/grafana/tree/main/.github/workflows/auto-triager

My previous PR put the wrong path.